soc/ideas-page.md

```diff
@@ -329,9 +331,6 @@ Julia employs several techniques that are novel in the field of high performance
 
 # Theme: Julia Graphics and User Interfaces
 
-## 2D Graphics Improvements
```

Comment:
Oh yes. My bad. I was thinking of putting in a PR for this. Didn't realize I had this on the same branch. Thank you catching this.